CommandHelper/LocalPackages/commands/chars/madhatterjervistetch.yml

```diff
@@ -0,0 +1,29 @@
+class: madhatterjervistetch
+costumes:
+- chatcolor: '&e'
+  name:  MadHatter(JT)
```

Comment:
To add colour to characters in game, you use &(Number/Letter), in here, you have to replace the & with '§', and you use it the exact same but just replace them.